test/basic.html

```diff
@@ -29,9 +29,9 @@
       <div>
         <iron-form id="native-checkboxes">
           <form action="/get" method="get">
-            <input type="checkbox" name="check1" checked> <!-- default value is "on" -->
-            <input type="checkbox" name="check1" value="1">
-            <input type="checkbox" name="check1" value="2" checked>
+            <input type="checkbox" name="check1[]" checked> <!-- default value is "on" -->
```

Comment:
I believe I did this as a convention for feature detection on array-typed properties, as standard form does. Don't remember exactly the value of this, will have to revisit code.